code/game/machinery/big_manipulator.dm

```diff
@@ -234,6 +390,64 @@
 /obj/machinery/big_manipulator/proc/finish_rotate_animation(backward)
 	animate(manipulator_hand, transform = matrix(180 * backward, MATRIX_ROTATE), working_speed*0.5)
 
+/obj/machinery/big_manipulator/proc/check_filter(obj/item/what_item)
+	var/filtered_obj = filter_obj?.resolve()
+	if(!istype(what_item, selected_type))
+		return
+	/// We use filter only on items. closets, humans and etc don't need filter check.
+	if(istype(what_item, /obj/item))
+		if((filtered_obj && !istype(what_item, filtered_obj)))
+			return FALSE
+	return TRUE
+
+/// Create dummy to force him use our item and then delete him.
+/obj/machinery/big_manipulator/proc/create_abstract_living()
+	var/mob/living/carbon/human/abstract_living = new /mob/living/carbon/human(get_turf(src))
```

Comment:
Icky about this, however:

1) This should be a human/dummy, which is essentially immune to several ailments and quirky things that can happen as result of using some items.
2) This mob should have its mind initialized. I'm sure that there are some objects in the code that assume the user of an item has a mind, and perhaps a client. While we can do little for clients afaik,  they can still have clientless mind datums.

Reply:
You mean to make sure that mind is not null?

Reply:
Or you already did everything for me when you added mind_initialize